### ISSUE TITLE
Save button possibly missing after Sacred Nebula?

### ISSUE THREAD

**Issue by sserwatka993** [Redcrafter/lbr-save-editor#6]
So, I'm not sure whether this is because I'm currently past the first zone and in Sacred Nebula, but since getting there, I've noticed that the save button no longer appears upon loading the file to edit.

Unsure on whether this is intentional or not to avoid corrupted saves.

**Comment by Redcrafter** [comment]
Could you provide me a save file where that happens?

**Comment by coolmule0** [comment]
I have the same issue. The `saveButton` element has a `hidden=true` setting. If i remove this from the css, the button appears and I can download the save without issue.
In the console I get an error:
```
Uncaught TypeError: str is undefined
    pretty https://redcrafter.github.io/lbr-save-editor/index.js:612
    loadCraftedLeaves https://redcrafter.github.io/lbr-save-editor/index.js:583
    createTables https://redcrafter.github.io/lbr-save-editor/index.js:552
    processSave https://redcrafter.github.io/lbr-save-editor/index.js:378
    loadDat https://redcrafter.github.io/lbr-save-editor/index.js:704
    <anonymous> https://redcrafter.github.io/lbr-save-editor/index.js:255
    EventListener.handleEvent* https://redcrafter.github.io/lbr-save-editor/index.js:252
    EventListener.handleEvent* https://redcrafter.github.io/lbr-save-editor/index.js:250
    EventListener.handleEvent* https://redcrafter.github.io/lbr-save-editor/index.js:247
index.js:612:15
    pretty https://redcrafter.github.io/lbr-save-editor/index.js:612
    loadCraftedLeaves https://redcrafter.github.io/lbr-save-editor/index.js:583
    createTables https://redcrafter.github.io/lbr-save-editor/index.js:552
    processSave https://redcrafter.github.io/lbr-save-editor/index.js:378
    loadDat https://redcrafter.github.io/lbr-save-editor/index.js:704
    <anonymous> https://redcrafter.github.io/lbr-save-editor/index.js:255
    (Async: EventListener.handleEvent)
    <anonymous> https://redcrafter.github.io/lbr-save-editor/index.js:252
    (Async: EventListener.handleEvent)
    <anonymous> https://redcrafter.github.io/lbr-save-editor/index.js:250
    (Async: EventListener.handleEvent)
    <anonymous> https://redcrafter.github.io/lbr-save-editor/index.js:247
```
Attached a zip of a save file.
[save.dat.zip](https://github.com/Redcrafter/lbr-save-editor/files/11504602/save.dat.zip)

**Comment by Redcrafter** [comment]
Thanks for the save file. Fixed with e20dc7a
